api/app/controllers/spree/api/v2/base_controller.rb

```diff
@@ -13,6 +13,10 @@ def render_serialized_payload(payload, status)
           render json: { error: exception.message }, status: 400
         end
 
+        def render_error_payload(error, status = 422)
```

Comment:
Looks good, but can you use this method in base class, as I see there is still rendering errors with regular syntax

Reply:
@VygovskySergey good catch!